app/views/shared/_media_objects_view.html.erb

```diff
@@ -7,6 +7,7 @@
         <th style="max-width:10%">File</th>
         <th style="max-width:60%">Name</th>
         <th style="max-width:10%"></th>
+        <th style="max-width:10%"></th>
```

Comment:
Actually @jaypoulz I did add another td, you sure this isn't okay? I'm counting 5 tds and 5 ths